### PR TITLE
Update strings_fr.json

### PR DESCRIPTION
**Pull request by balbuze**
add new fields

### CODE REVIEW

File: app/i18n/strings_fr.json [hunk 1]
```diff
@@ -73,7 +73,15 @@
     "EXTENDED_FILE_FORMAT_SUPPORT_DOC":"Active la possibilité de lire de plus nombreux formats audio. Cependant, cela ralentira la création de la bibliothèque musicale. Si désactivé Volumio pourra toujours lire la pluplart des fichiers et la création sera plus rapide.",
     "USB_DAC_CONNECTED":"Dac USB DAC Connecté",
     "NO_OUTPUT_DEVICE":"Pas de sortie audio disponible",
-    "NO_OUTPUT_DEVICE_MESSAGE":"Connecté un péréphérique audio à ce matériel"
+    "NO_OUTPUT_DEVICE_MESSAGE":"Connectez un péréphérique audio à ce matériel",
```
Comment (JonathanHuot) [inline app/i18n/strings_fr.json]: péréphérique -> périphérique

File: app/i18n/strings_fr.json [hunk 2]
```diff
@@ -73,7 +73,15 @@
...
+    "OUTPUT_DEVICE_NOT_AVAILABLE":"La sortie audio chooisie n'est pas disponible",
```
Comment (JonathanHuot) [inline app/i18n/strings_fr.json]: chooisie -> choisie

File: app/i18n/strings_fr.json [hunk 3]
```diff
@@ -73,7 +73,15 @@
...
+    "CONNECT_OUTPUT_DEVICE_1":"Connectez",
+    "CONNECT_OUTPUT_DEVICE_2":"au périphérique",
+    "CONNECT_OUTPUT_DEVICE_2":"au périphérique ou selectionnez une autre sortie audio",
+    "PLAYBACK_MODE":"Playback Mode",
+    "PLAYBACK_MODE_DOC":"Vous pouvez choisir le comportement de lecture : Continu remplacera la liste courante par tous les morceaux de l'album ou de la playlist, Unitaire ajoute seulement le morceau selectionné",
```
Comment (JonathanHuot) [inline app/i18n/strings_fr.json]: Continue -> Continue

File: app/i18n/strings_fr.json [hunk 4]
```diff
@@ -73,7 +73,15 @@
...
+    "PLAYBACK_MODE_CONTINUOUS":"Continu",
```
Comment (JonathanHuot) [inline app/i18n/strings_fr.json]: Continue -> Continue

File: app/i18n/strings_fr.json [hunk 5]
```diff
@@ -313,15 +323,22 @@
     "FULL_SETTINGS_SET":"Mode réglage expert",
     "FULL_SETTINGS_SET_EXTENDED":"Accès à toutes les options de réglage",
     "SIMPLE_SETTINGS_SET":"Mode réglage simplifié",
-    "SIMPLE_SETTINGS_SET_EXTENDED":"Accès seulement aux réglages de base"
+    "SIMPLE_SETTINGS_SET_EXTENDED":"Accès seulement aux réglages de base",
+    "KEEP_PRESSED_TO_TURN_OFF_IN":"Garder appuyer pour éteindre",
```
Comment (JonathanHuot) [inline app/i18n/strings_fr.json]: Garder -> Gardez

File: app/i18n/strings_fr.json [hunk 6]
```diff
@@ -313,15 +323,22 @@
...
   },
   "WEBRADIO":{
     "MY_WEB_RADIOS":"Mes Web Radios",
     "FAVOURITE_RADIOS":"Radios Favorites",
     "TOP_500_RADIOS":"Top 500 Radios",
     "BY_GENRE_RADIOS":"Par genre",
     "BY_COUNTRY_RADIOS":"Par pays",
-    "WEBRADIO":"Web Radio"
+    "WEBRADIO":"Web Radio",
+    "LOCAL_RADIOS":"Radios Locales",
```
Comment (JonathanHuot) [inline app/i18n/strings_fr.json]: Radios Locales -> Radios locales

File: app/i18n/strings_fr.json [hunk 7]
```diff
@@ -313,15 +323,22 @@
...
+    "TURN_OFF_IN":"Eteindre"  
   },
   "WEBRADIO":{
     "MY_WEB_RADIOS":"Mes Web Radios",
     "FAVOURITE_RADIOS":"Radios Favorites",
     "TOP_500_RADIOS":"Top 500 Radios",
     "BY_GENRE_RADIOS":"Par genre",
     "BY_COUNTRY_RADIOS":"Par pays",
-    "WEBRADIO":"Web Radio"
+    "WEBRADIO":"Web Radio",
```
Comment (JonathanHuot) [inline app/i18n/strings_fr.json]: Web Radio -> Radio Internet

File: app/i18n/strings_fr.json [hunk 8]
```diff
@@ -313,15 +323,22 @@
...
+    "LOCAL_RADIOS":"Radios Locales",
+    "POPULAR_RADIOS":"Radios populaires",
+    "BEST_RADIOS":"Meilleures Radios",
+    "BY_LANGUAGE":"Par Langue",
+    "TALK_RADIOS":"Radios vocales"
```
Comment (JonathanHuot) [inline app/i18n/strings_fr.json]: Radios vocales -> Emissions radios ?

**Comment by balbuze** [comment]
thanks @JonathanHuot

File: app/i18n/strings_fr.json [hunk 9]
```diff
@@ -73,7 +73,15 @@
...
     "USB_DAC_CONNECTED":"Dac USB DAC Connecté",
     "NO_OUTPUT_DEVICE":"Pas de sortie audio disponible",
-    "NO_OUTPUT_DEVICE_MESSAGE":"Connecté un péréphérique audio à ce matériel"
+    "NO_OUTPUT_DEVICE_MESSAGE":"Connectez un périphérique audio à ce matériel",
+    "OUTPUT_DEVICE_NOT_AVAILABLE":"La sortie audio choisie n'est pas disponible",
+    "CONNECT_OUTPUT_DEVICE_1":"Connectez",
+    "CONNECT_OUTPUT_DEVICE_2":"au périphérique",
+    "CONNECT_OUTPUT_DEVICE_2":"au périphérique ou selectionnez une autre sortie audio",
+    "PLAYBACK_MODE":"Playback Mode",
+    "PLAYBACK_MODE_DOC":"Vous pouvez choisir le comportement de lecture : Continue remplacera la liste courante par tous les morceaux de l'album ou de la playlist, Unitaire ajoute seulement le morceau selectionné",
```
Comment (JonathanHuot) [inline app/i18n/strings_fr.json]: ajoute -> ajoutera

File: app/i18n/strings_fr.json [hunk 10]
```diff
@@ -313,15 +323,22 @@
     "FULL_SETTINGS_SET":"Mode réglage expert",
     "FULL_SETTINGS_SET_EXTENDED":"Accès à toutes les options de réglage",
     "SIMPLE_SETTINGS_SET":"Mode réglage simplifié",
-    "SIMPLE_SETTINGS_SET_EXTENDED":"Accès seulement aux réglages de base"
+    "SIMPLE_SETTINGS_SET_EXTENDED":"Accès seulement aux réglages de base",
+    "KEEP_PRESSED_TO_TURN_OFF_IN":"Gardez appuyer pour éteindre",
```
Comment (JonathanHuot) [inline app/i18n/strings_fr.json]: Gardez appuyer -> Maintenir appuyé

**Comment by JonathanHuot** [comment]
sorry @balbuze, I spotted other two issues :D :D that should be good now! good job

**Comment by balbuze** [comment]
@JonathanHuot : TY!

**Comment by balbuze** [comment]
Yes!

**Pull request merged**